src/quacc/recipes/qchem/ts.py

```diff
@@ -229,3 +230,81 @@ def quasi_irc_job(
     relax_summary["initial_irc"] = irc_summary
 
     return relax_summary
+
+
+@job
+@requires(Sella, "Sella must be installed. Refer to the quacc documentation.")
+def quasi_irc_perturb_job(
+    atoms: Atoms,
+    charge: int,
+    spin_multiplicity: int,
+    mode: list[list[float]],
+    perturb_magnitude: float = 0.6,
+    direction: Literal["forward", "reverse"] = "forward",
+    method: str = "wb97mv",
+    basis: str = "def2-svpd",
+    opt_params: dict[str, Any] | None = None,
+    copy_files: SourceDirectory | dict[SourceDirectory, Filenames] | None = None,
+    **calc_kwargs,
+) -> OptSchema:
+    """
+    Quasi-IRC to optimize a reaction endpoint from a transition-state with known vibrational frequency modes.
+    Perturbs the structure of `atoms` by a finite amount (0.6 * the normalized mode magnitude) along the specified
+    vibrational frequency mode (assumed to be the transition mode), and then performs a `relax_job` on the perturbed
+    structure.
+
+    Parameters
+    ----------
+    atoms
+        Atoms object.
+    charge
+        Charge of the system.
+    spin_multiplicity
+        Multiplicity of the system.
+    mode
+        Transition mode
+    perturb_magnitude
+        Factor to multiply the transition mode. Default is 0.6. In some cases, it may be advisable to increase this
+        factor, perhaps to 1.0 or 1.1. Lowering it is not generally found to be helpful.
+    direction
+        Direction of the (Quasi)IRC. Should be "forward" or "reverse".
+    method
+        DFT exchange-correlation functional or other electronic structure
+        method.
+    basis
+        Basis set.
+    opt_params
+        Dictionary of custom kwargs for the optimization process. For a list
+        of available keys, refer to [quacc.runners.ase.run_opt][].
+    copy_files
+        Files to copy (and decompress) from source to the runtime directory.
+    **calc_kwargs
+        Custom kwargs for the calculator. Set a value to `quacc.Remove` to remove
+        a pre-existing key entirely. See [quacc.calculators.qchem.qchem.QChem][] for more
+        details.
+
+
+    Returns
+    -------
+    OptSchema
+        Dictionary of results from [quacc.schemas.ase.summarize_opt_run][]
+    """
+
+    calc_defaults = recursive_dict_merge(
+        _BASE_SET, {"rem": {"job_type": "force", "method": method, "basis": basis}}
+    )
+    opt_defaults = {"optimizer": Sella} if (Sella is not False) else {}
```

Comment:
Mind if we change this to:

```python
{"optimizer": Sella} if has_sella else {}
```

to match what's in `core.py`? This would involve having 

```python
try:
    from sella import IRC, Sella

except ImportError:
    Sella = False
```

changed to

```python
try:
    from sella import IRC, Sella
    has_sella = True

except ImportError:
    has_sella = False

Reply:
Yeah, that seems fine. Mostly I just didn't want to mess with existing code if I didn't have to. New repo for me, and all that.